### PR TITLE
Feat/anthos storage cta

### PR DESCRIPTION
**Pull request by ganeshkumar1989**
| Question         | Answer
| ---------------- | ---
| Branch?          | `feat/anthos-storage`
| Bug fix?         | no
| New feature?     | yes
| Breaking change? | no
| Tickets          | Fix #MANAGER-7393
| License          | BSD 3-Clause

## Description

Add "Add SVM Volume" and "Delete" CTAs to the Anthos storage page

### CODE REVIEW

File: packages/manager/modules/anthos/src/dashboard/components/storage-usage/controller.js [hunk 1]
```diff
@@ -0,0 +1,27 @@
+export default class AnthosStorageUsageCtrl {
+  /* @ngInject */
+  constructor($translate) {
+    this.$translate = $translate;
+  }
+
+  static mbToTb(value) {
+    return value / 1024 / 1024;
```
Comment (marie-j) [inline packages/manager/modules/anthos/src/dashboard/components/storage-usage/controller.js:8]: Use a  constant maybe ?

File: packages/manager/modules/anthos/src/dashboard/components/storage-usage/translations/Messages_fr_FR.json [hunk 2]
```diff
@@ -0,0 +1,5 @@
+{
+  "anthos_dashboard_storage_usage_system": "Systeme",
```
Comment (marie-j) [inline packages/manager/modules/anthos/src/dashboard/components/storage-usage/translations/Messages_fr_FR.json:2]:
```suggestion
  "anthos_dashboard_storage_usage_system": "Système",
```

File: packages/manager/modules/anthos/src/dashboard/routing.js [hunk 3]
```diff
@@ -3,10 +3,33 @@ export default /* @ngInject */ ($stateProvider) => {
...
       currentActiveLink: /* @ngInject */ ($transition$, $state) => () =>
         $state.href($state.current.name, $transition$.params()),
       dashboardLink: /* @ngInject */ ($state, serviceName) =>
         $state.href('anthos.dashboard', { serviceName }),
+      goToState: ($state, Alerter, alertId) => (
+        state,
+        stateParams = {},
+        message = false,
+        type = 'done',
+      ) => {
```
Comment (marie-j) [inline packages/manager/modules/anthos/src/dashboard/routing.js:16]: I'm not sure what the purpose of this is but controllers shouldn't have to worry to which state they go, they should just `goBack` or `goToDashboard` for example

File: packages/manager/modules/anthos/src/dashboard/storage/remove/template.html [hunk 4]
```diff
@@ -0,0 +1,31 @@
+<oui-modal
+    data-primary-action="$ctrl.removeStorage()"
+    data-primary-label="{{ ::'anthos_dashboard_remove_storage_submit' | translate }}"
+    data-primary-disabled="$ctrl.terminateConfirmation !== 'TERMINATE'"
```
Comment (marie-j) [inline packages/manager/modules/anthos/src/dashboard/storage/remove/template.html:4]: TERMINATE could be stored as a constant, Also if we're using an input we could benefit from `ng-pattern` instead

File: packages/manager/modules/anthos/src/dashboard/storage/remove/translations/Messages_fr_FR.json [hunk 5]
```diff
@@ -0,0 +1,10 @@
+{
+  "anthos_dashboard_remove_storage_title": "Supprimer SVM",
+  "anthos_dashboard_remove_storage_description": "Vous supprimerez définitivement <strong>{{storageName}}</strong>. Vous perdrez tout le contenu.",
+  "anthos_dashboard_remove_storage_warning": "A La suppression entrainera l'effacement définitif de toutes les données correspondantes.",
```
Comment (marie-j) [inline packages/manager/modules/anthos/src/dashboard/storage/remove/translations/Messages_fr_FR.json:4]: Content looks odd, was this the one provided ?

File: packages/manager/modules/anthos/src/dashboard/components/storage-usage/controller.js (hunk 1 above)
Comment (ganeshkumar1989) [reply packages/manager/modules/anthos/src/dashboard/components/storage-usage/controller.js]: Done :)

File: packages/manager/modules/anthos/src/dashboard/components/storage-usage/translations/Messages_fr_FR.json (hunk 2 above)
Comment (ganeshkumar1989) [reply packages/manager/modules/anthos/src/dashboard/components/storage-usage/translations/Messages_fr_FR.json]: Fixed

File: packages/manager/modules/anthos/src/dashboard/routing.js (hunk 3 above)
Comment (ganeshkumar1989) [reply packages/manager/modules/anthos/src/dashboard/routing.js:16]: This is a kind of a reusable method, which avoids code replication in many places where we write the `goTo*` functions

File: packages/manager/modules/anthos/src/dashboard/storage/remove/template.html (hunk 4 above)
Comment (ganeshkumar1989) [reply packages/manager/modules/anthos/src/dashboard/storage/remove/template.html]: This is done

File: packages/manager/modules/anthos/src/dashboard/storage/remove/translations/Messages_fr_FR.json (hunk 5 above)
Comment (ganeshkumar1989) [reply packages/manager/modules/anthos/src/dashboard/storage/remove/translations/Messages_fr_FR.json:4]: Yeah... it was provided in specs. Probably need a review

**Comment by mohammed-zahaf** [comment]
Unblocked, content has been provided and updated

File: packages/manager/modules/anthos/src/dashboard/storage/add/controller.js [hunk 6]
```diff
@@ -0,0 +1,27 @@
+export default class {
+  /* @ngInject */
+  constructor($translate, AnthosTenantsService) {
+    this.$translate = $translate;
+    this.AnthosTenantsService = AnthosTenantsService;
+  }
+
+  addStorage() {
```
Comment (varun257) [inline packages/manager/modules/anthos/src/dashboard/storage/add/controller.js:8]: `loading` missing here and in modal?

Comment (mohammed-zahaf) [reply packages/manager/modules/anthos/src/dashboard/storage/add/controller.js:8]: Yep true.
Updated, thanks

**Review by radireddy — APPROVED** [review]

**Review by mohammed-zahaf — APPROVED** [review]

**Pull request merged**